vue.config.js

```diff
@@ -4,6 +4,13 @@ module.exports = {
   devServer: {
     port: 3002
   },
+  chainWebpack: config => {
```

Comment:
Hey @RakowskiPrzemyslaw, 

great to see that you have picked my proposal from #45. 

One little not here: You have to change the config, to use webpacks `oneOf` option. Because replacing the complete `file-loader` for SVGs with the `vue-svg-loader` will cause something like `<img src=[Object Module] >/` when SVG used in a img tag (the logo for example). 

A working config could look like this: 

```js
chainWebpack: config => {
    const svgRule = config.module.rule("svg");

    svgRule.use("file-loader").tap(options => {
      originalOptions = options;
    });

    svgRule.uses.clear();

    svgRule
      .oneOf("svg-component")
      .resourceQuery(/\?component/)
      .use("vue-svg-loader")
      .loader("vue-svg-loader")
      .end()
      .end()
      .oneOf("normal")
      .use("file-loader")
      .loader("file-loader")
      .options(originalOptions);
  }
```

With this config, a `import SfSvgAlert from '/assets/alert-alert.svg?inline';` (the important part is the resource query) will be loaded as a Vue component and `<img src="../../../logo.svg />` will use the normal `file-loader` as aspected.